packages/@o3r/core/schematics/ng-add/utils/linter/index.ts

```diff
@@ -0,0 +1,25 @@
+import { SchematicContext } from '@angular-devkit/schematics';
+import { askConfirmation } from '@angular/cli/src/utilities/prompt';
+
+export const shouldOtterLinterBeInstalled = async (context: SchematicContext): Promise<boolean> => {
+  const linterPackageName = 'eslint';
+  let useOtterLinter = false;
+  try {
+    require.resolve(`${linterPackageName}/package.json`);
+    if (context.interactive) {
+      useOtterLinter = await askConfirmation(`You already have eslint installed. Would you like to add otter config rules for eslint ?
+Otherwise, you can add them later via this command: ng add @o3r/eslint-config-otter`, true);
+      if (!useOtterLinter) {
+        context.logger.info(`
+          You can add otter linter config rules later to the project via this command:
+          ng add @o3r/eslint-config-otter
```

Comment:
we already say it just above 
do we really need to say it again ?

Reply:
Indeed I can remove the check and this log. It will be simpler